packages/core/src/Site/index.js

```diff
@@ -889,6 +927,42 @@ class Site {
     });
   }
 
+  /**
+   * Handles the reloading of ignore attributes
+   */
+  async handleIgnoreReload(oldSiteConfig) {
+    const assetsToRemove = _.difference(this.siteConfig.ignore, oldSiteConfig.ignore);
+
+    if (!_.isEqual(oldSiteConfig.ignore, this.siteConfig.ignore)) {
+      await this._removeMultipleAssets(assetsToRemove);
+      this.rebuildManagers(oldSiteConfig);
+      await this.buildAssets();
+    }
+  }
+
+  /**
+   * Chceks the relevant site configurations for changes and rebuild managers if necessary
+   */
+  rebuildManagers(oldSiteConfig) {
+    if (!_.isEqual(oldSiteConfig.ignore, this.siteConfig.ignore)
+      || !_.isEqual(oldSiteConfig.plugins, this.siteConfig.plugins)
+      || !_.isEqual(oldSiteConfig.pluginsContext, this.siteConfig.pluginsContext)
+      || !_.isEqual(oldSiteConfig.disableHtmlBeautify, this.siteConfig.disableHtmlBeautify)
+      || !_.isEqual(oldSiteConfig.intrasiteLinkValidation, this.siteConfig.intrasiteLinkValidation)) {
```

Comment:
missing `timeZone / locale / style` as well - should / could we just remove the checks here as well?
I feel there's not much value in further narrowing it down since `buildManagers` isn't very costly

Reply:
> I feel there's not much value in further narrowing it down since buildManagers isn't very costly

Alright since there's not much value for the checks as it is not costly, then `buildManagers` will be called for every full rebuild and whenever `ignore` is modified. I've updated the PR accordingly.